PLATFORMS.md

```diff
@@ -0,0 +1,64 @@
+As of 0.9, the following platforms are supported:
```

Comment:
I think we should remove the reference to a specific platform (So just say "Currently, the following.." or so), otherwise this is yet another place where we have to update a version string, and the version should be clear from the OF release this document is delivered with.